sw/device/silicon_creator/lib/base/sec_mmio.h

```diff
@@ -109,6 +109,30 @@ OT_ASSERT_MEMBER_OFFSET(sec_mmio_ctx_t, expected_write_count, 1608);
 OT_ASSERT_MEMBER_OFFSET(sec_mmio_ctx_t, check_count, 1612);
 OT_ASSERT_SIZE(sec_mmio_ctx_t, 1616);  // Checked by linker script.
 
+// The sec_mmio_ctx is referenced here to be able to use it inside the
+// `SEC_MMIO_WRITE_INCREMENT()` macro.
+extern volatile sec_mmio_ctx_t sec_mmio_ctx;
```

Comment:
How do we know this can't race? I.e. what happens if this gets touched by an ISR? You could wind up with a situation like:
```S
lw   a0, sec_mmio_ctx
// Interrupt fired here.
addi a0, a0, kIncrement
sw   a0, sec_mmio_ctx
```
Any writes made to it by the ISR will be discarded.

Given the lack of atomic RMW operations, I'm not sure how we can make this thread-safe, except by disabling
interrupts whenever this variable gets touched. That won't stop NMIs, of course... =/